packages/core/src/state-management/state-manager.ts

```diff
@@ -196,7 +197,8 @@ export class StateManager {
           }
         }),
         catchError((error) => {
-          this.logger.err(error);
+          this.logger.err(`Error while anchoring stream ${state$.id.toString()}: ${error.message}`)
+          this.logger.err(error); // Log full error to get the stack trace
```

Comment:
Probably better to put the stack trace in the same logger call, as these are not guaranteed to be sequential

```js
this.logger.err(`Error while anchoring stream ${state$.id.toString()}: ${error.message}\n${error}`)
```